samples/test-samples.cmake

```diff
@@ -24,14 +24,27 @@ else ()
   # These tests can only run with SGX-FLC, meaning they were built
   # against SGX.
   if (USE_LIBSGX)
-    list(APPEND SAMPLES_LIST local_attestation remote_attestation attested_tls)
+    list(APPEND SAMPLES_LIST local_attestation remote_attestation)
+	# The attested_tls test is not supported on Windows at this time.
+	if (NOT WIN32)
+	  list(APPEND SAMPLES_LIST attested_tls)
+	endif ()
   endif ()
 endif ()
 
-execute_process(COMMAND ${CMAKE_COMMAND} -E env DESTDIR=${BUILD_DIR}/install ${CMAKE_COMMAND} --build ${BUILD_DIR} --target install)
+if (WIN32)
+  # On Windows, DESTDIR is not supported by cmake for the install function. Must use a relative path for -DCMAKE_INSTALL_PREFIX:PATH
+  execute_process(COMMAND ${CMAKE_COMMAND} --build ${BUILD_DIR} --target install)
+else ()
+  execute_process(COMMAND ${CMAKE_COMMAND} -E env DESTDIR=${BUILD_DIR}/install ${CMAKE_COMMAND} --build ${BUILD_DIR} --target install)
+endif ()
 
-# The prefix is appended to the value given to DESTDIR, e.g. build/install/opt/openenclave/...
-set(INSTALL_DIR ${BUILD_DIR}/install${PREFIX_DIR})
+if (WIN32)
```

Comment:
Can we combine this if-else clause with the previous since the two are related to the DESTDIR handling differences anyway?

Reply:
Yes, good point!